pom.xml

```diff
@@ -1676,6 +1676,44 @@ under the License.
 							<version>1.11.1</version>
 						</dependency>
 					</dependencies>
+					<configuration>
+						<ignoredUsedUndeclaredDependencies combine.children="append">
+							<!-- allow using transitive Flink dependencies for brevity -->
+							<dependency>org.apache.flink:*</dependency>
```

Comment:
Is there a shortcut to ignore dependencies in `test` scope?

Reply:
not sure, but I would advice against it anyway since unused test dependencies are also problematic form a maintenance perspective as the still restrict the dependency graph (in case of Flink dependencies) and may introduce dependency conflicts.

Reply:
fair enough